docs/architecture.md

```diff
@@ -1,103 +1,172 @@
+
+The listener service is used to detect your voice.  It controlls the WakeWord, STT (Speech To Text), and VAD Plugins.  You can modify microphone settings and enable additional features under the listener section such as wake word / utterance recording / uploading
+
+The [ovos-dinkum-listener](https://github.com/OpenVoiceOS/ovos-dinkum-listener) is the new OVOS listener that replaced the origional [ovos-listener](https://github.com/OpenVoiceOS/ovos-listener) and has many more options.  Others still work, but are not recommended.
+
+[Technical Listener Docs](https://openvoiceos.github.io/ovos-technical-manual/speech_service/)
+
+typical systemd command
+
+`systemctl --user start ovos-dinkum-listener`
+
+### STT Plugins
+
+This is where speech is transcribed into text and forwarded to the skills service.
+
+Two STT plugins may be loaded at once, if the primary plugin fails for some reason the second plugin will be used.
+
+This allows you to have a lower accuracy offline model as fallback to account for internet outages, this ensures your device never becomes fully unusable.
+
+Several different STT (Speech To Text) Plugins are avaliable for use.  OVOS provides a number of public servers using the [ovos-stt-plugin-server](https://github.com/OpenVoiceOS/ovos-stt-plugin-server) plugin and are hosted by OVOS contributors.  No additional configuration is required.
+
+[OVOS Supported STT Plugins](https://github.com/orgs/OpenVoiceOS/repositories?q=ovos-stt-plugin&type=all&language=&sort=)
+
+**TODO** Fix link to how to change stt
+
+[Changing STT Plugin](stt-plugins.md)
+
+### WakeWord Plugins
 
-### Message
-A Message consists of a json payload, it contains a `type` , some `data` and a `context`. 
+A WakeWord is what OVOS uses to activate the device.  By default `Hey Mycroft` is used by OVOS.  Like other things in the OVOS ecosystem, this is configurable.
 
-The `context` is considered to be metadata and might be changed at any time in transit, the `context` can contain anything depending on where the message came from, and often is completely empty. 
+[OVOS Supported WakeWord Plugins](https://github.com/orgs/OpenVoiceOS/repositories?q=ovos-ww-plugin&type=all&language=&sort=)
 
-You can think of the message `context` as a sort of session data for a individual interaction, in general messages down the chain keep the `context` from the original message, most listeners (eg, skills) will only care about `type` and `data`. 
+**TODO** Fix link to change ww
 
-ovos-core uses the message `context` to add metadata about the messages themselves, where do they come from and what are they intended for.
+[Changing the WakeWord](change-ww.md)
 
-#### Sources
+#### HotWords
 
-ovos-core injects the context when it emits an utterance, this can be either typed in the `ovos-cli-client` or spoken via STT service
+OVOS allows you to load any number of hot words in parallel and trigger different actions when they are detected
```

Comment:
```suggestion
OVOS allows you to load any number of hot words in parallel and trigger different actions when they are detected.
```

Reply:
Consider explaining the difference between a wake word and a hot word.

Reply:
Have a suggestion on that?  They differ some, I think, but not sure how to explain it

Reply:
everything is a hotword, wake words are hotwords that trigger listening

a hotword can also play a sound or emit an event for some other purpose that is not listening

theres also 2 more special cases of hotword in the listener, wakeup_word to get out of sleep mode, and stopword to get out of recording mode